demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -168,7 +168,7 @@ public void initiliazeProjectProductData() {
                     .project(project1)
                     .description("Test build config for project newcastle")
                     .environment(environment1)
-                    .buildScript("mvn clean deploy -Dmaven.test.skip")
+                    .buildScript("mvn clean deploy -DskipTests=true")
```

Comment:
Isn't it better for faster build to skip also the compilation of tests and not just the execution ?